interface/main/calendar/modules/PostCalendar/pnadmin.php

```diff
@@ -441,18 +440,18 @@ function postcalendar_admin_categories($msg = '', $e = '', $args = array())
         $template_name ='default';
     }
 
-    $output->Text(postcalendar_adminmenu());
+    $output->Text(postcalendar_adminmenu("category"));
 
     if (!empty($e)) {
-        $output->Text('<div style="padding:5px; border:1px solid red; background-color: pink;">');
-        $output->Text('<center><b>' . text($e) . '</b></center>');
-        $output->Text('</div><br />');
+        $output -> Text('<div class="alert alert-danger ml-5 mr-5" role="alert">');
+        $output->Text('<b>'. text($e) .'</b>');
+        $output -> Text('</div><br />');
     }
 
     if (!empty($msg)) {
-        $output->Text('<div style="padding:5px; border:1px solid green; background-color: lightgreen;">');
-        $output->Text('<center><b>' . text($msg) . '</b></center>');
-        $output->Text('</div><br />');
+        $output -> Text('<div class="alert alert-success ml-5 mr-5" role="alert">');
+        $output->Text('<b>' . text($msg) . '</b>');
```

Comment:
This should be `<span class="text-center font-weight-bold">` instead of using the outdated `<b>` tag. You also forgot to address the `<center>` tag that was previously here.